spec/rails_helper.rb

```diff
@@ -57,9 +57,10 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
-
-  config.include Devise::Test::ControllerHelpers, type: :controller
-  config.include Devise::Test::ControllerHelpers, type: :view
-  config.include Devise::Test::IntegrationHelpers, type: :request
-  config.include Devise::Test::IntegrationHelpers, type: :feature
+  [:controller, :view].each do |type|
+    config.include Devise::Test::ControllerHelpers, type:
+  end
+  [:feature, :request, :system].each do |type|
```

Comment:
Nice!  I don't think we have any feature specs in this project, though.  Could you remove `:feature` while you're at it?
```suggestion
  [:request, :system].each do |type|
```

Reply:
Done!